src/commands/command-getkeysandflags.json

```diff
@@ -14,6 +14,17 @@
         ],
         "acl_categories": [
             "CONNECTION"
+        ],
+        "arguments": [
+            {
+                "name": "command",
+                "type": "string"
+            },
+            {
+                "name": "arg",
+                "type": "string",
+                "multiple": true
```

Comment:
same, add `optional`

Reply:
I found this commands was added in 7.0, so we need not optional?

Reply:
i still think it makes sense to call it on PING.
it's not just because of the regression in 7.0, it's because it makes sense to cal it on whatever command you want.

please fix the arity on both commands we well.